projects/Douglas Mentoring/day1.py

```diff
@@ -0,0 +1,92 @@
+import getpass
+import netmiko
+# json was here bc as I'm using textfsm to pull data from cisco devices
+# it comes in json i needed to show the output before deciding how to tread the date
+import json
+import csv
+
+# Where i'm pulling the devices list / Soon I'll make this automated which will pull direct from NARF
+file = open(r'/home/segattod/segattod-workspace/src/Segattod-package/myproject/Light_level_checks/list.txt',
+            'r')
+file = file.read().splitlines()
+
+username = input('User Name:')
+password = getpass.getpass()
+
+# Variable for "append" the data later
+formatted_data = []
+
+# for now just to name the file / Soon to add for user to add input :)
+whid = 'YEG1'
```

Comment:
By convention, names of constants are always in all caps
```suggestion
WHID = 'YEG1'
```

Reply:
okay, even if later I'll make it to ask user the "whid "input, I should make it all capital to mean it should not changed, right?

Reply:
I'm actually talking about that in the other file I'm going to upload.  it's an interesting one.  In general, the value should never change so I'd say yes, all caps.  BUT from the other point of view, it IS changing once because it is being set.  In the end, it could go either way, I guess.